### DISCUSSION TITLE
Apollo Save Tool v2.0.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#199] [Announcements]
### Added

* Complete PS4 firmware support (works on all FWs) - thanks to @hzhreal / @Team-Alua
  - Firmware agnostic
  - No need for GoldHEN, PS4HEN, or ps4debug payload
* Trophy management
  - List, mount, and export trophies
  - Fake XMB trophies (lock/unlock)
  - Trophy set export and backup
  - Remove trophy sets
* Delete PS4 save-games
* Show date/time of last save-game modification

---

> ⚠️  **Note:** deleting a trophy set removes critical trophy files from the user account, and trying to play the game again won't populate trophies for that user. 

> ⚠️ **Note:** Fake trophy unlocking does not modify secure trophy files; is only doing changes to the XMB database. If you play the game after applying modifications, fake results can be reset by the system.

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v2.0.0'>Apollo Save Tool v2.0.0</a>.</em>

**Comment by JhoserpT** [comment]
Hello Buccaneer, I hope you are well...

I was going to tell you that I had an error with the application, I did the normal .pkg installation, using the GOLD HEN debug settings, everything seemed to work so I wanted to make a backup.

But when I searched for the saves on my HDD, only the Apollo save appeared, all my saves games were deleted I was just going to tell you about that error Bucanero, I wouldn't know how to help with the development anyway.

Greetings and have a nice day, I appreciate the excellent work 🙂‍↕️
